### DISCUSSION TITLE
During the script put the UI URL and Login info in the Notes for the container

### DISCUSSION THREAD

**Discussion by VoidLock** [tteck/Proxmox#3151] [Ideas]
During installation, when the script adds your logo and buy me a coffee, under that, put the relevant info to get to the UI as a hyperlink so the user doesn't have to go searching for what the port is set to or what the default login is.

**Comment by VoidLock** [comment]
![image](https://github.com/tteck/Proxmox/assets/37760961/3241343c-d4a9-4dbf-a3cf-f60694e6d675)

**Reply by Mellowlynx** [reply]
Nice idea to do it during the installation, might be a nice to have.
I do it manually now, inc. the local link, for when my proxy or ISP is down.

![image](https://github.com/tteck/Proxmox/assets/2348339/431c87b6-40e7-4ab3-8a8c-eb0b4accb215)

**Reply by di5cord20** [reply]
I do the same and agree it would be a nice to have.
